pom.xml

```diff
@@ -164,8 +190,10 @@
         <plugin.maven-gpg.version>1.6</plugin.maven-gpg.version>
         <plugin.maven-nexus-staging.version>1.6.7</plugin.maven-nexus-staging.version>
         <plugin.maven-site.version>3.7.1</plugin.maven-site.version>
-        <plugin.maven-scm-publish.version>1.1</plugin.maven-scm-publish.version>
+        <plugin.maven-scm-publish.version>3.0.0</plugin.maven-scm-publish.version>
         <plugin.maven-jacoco.version>0.8.1</plugin.maven-jacoco.version>
+
+        <maven.site.url>http://localhost/gradoop/site</maven.site.url>
```

Comment:
Localhost ?

Reply:
This was necessary to test the site locally, since site links are generated relative to this URL. I will remove this though, since this is not relevant anymore.